15-draft.json

```diff
@@ -816,6 +816,52 @@
             ]
           }
         },
+        "power_generation": {
+          "description": "The power generation of a specific device or of your whole space",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "value": {
+                "description": "The sensor value",
+                "type": "number"
+              },
+              "unit": {
+                "description": "The unit of the sensor value.",
+                "type": "string",
+                "enum": [
+                  "W",
+                  "VA"
```

Comment:
We removed `mW` since it doesn't make sense to have it, since `1e-3W` are equal to `1mW`. `VA` on the other hand has a slightly different meaning since it is used for [Apparent power](https://en.wikipedia.org/wiki/Apparent_power).

Reply:
We'll remove it from other sensors as well.